example/google.py

```diff
@@ -23,6 +23,9 @@
     consumer_secret=app.config.get('GOOGLE_SECRET'),
     request_token_params={
         'scope': 'https://www.googleapis.com/auth/userinfo.email'
+        # Note: multiple scopes are separated by spaces (not by comma)
```

Comment:
Yes, I used list and it worked for me. Here is example code:

request_token_params={
        'scope': [
             'https://mail.google.com/',
             'https://www.googleapis.com/auth/admin.directory.user.readonly',
             'https://www.googleapis.com/auth/admin.directory.orgunit.readonly',
             'https://www.googleapis.com/auth/admin.directory.group.readonly',
             'https://www.googleapis.com/auth/userinfo.email',
             'https://www.googleapis.com/auth/userinfo.profile',
             'https://www.googleapis.com/auth/plus.profile.emails.read',
 ]